src/components/organisms/UiModal/UiModal.vue

```diff
@@ -453,6 +455,7 @@ const titleAttrs = computed(() => (props.title
 const closeHandler = () => {
   if (!props.isClosable) return;
   emit('update:modelValue', false);
+  if (dialog.value) dialog.value.close();
```

Comment:
what is the reason to use native `.close`?  when you emit `emit('update:modelValue', false);` probably the component will be unmounted before this line

Reply:
you're right, this is a leftover